protocol/x/clob/keeper/liquidations.go

```diff
@@ -258,6 +244,10 @@ func (k Keeper) PlacePerpetualLiquidation(
 		metrics.PlacePerpetualLiquidation,
 	)
 
+	if err := k.validateLiquidationAgainstClobPairStatus(ctx, liquidationOrder); err != nil {
```

Comment:
Worth updating func comment that perp liquidations can return an error if placed on a non-active market?